collivery.php

```diff
@@ -8,7 +8,7 @@
  * Plugin Name: MDS Collivery
  * Plugin URI: https://collivery.net/integration/woocommerce
  * Description: Plugin to add support for MDS Collivery in WooCommerce.
```

Comment:
Im sure this version needed to match the version above, cant remember but have a look back at some of my commits for version change